frontend/src/components/details/top_card.svelte

```diff
@@ -44,17 +44,32 @@
     </div>
 
     {#if providers}
-        <div class="flex-nowrap pt-5">
-            {#each providers as provider}
-                <div class="avatar p-1">
-                    <div class="mb-8 w-20 h-20 mask mask-squircle">
+        {#if providers.length > 5}
+            <div class="grid grid-rows-2 grid-flow-col pt-5">
+                {#each providers as provider}
+                    <div class="mask mask-squircle p-1">
                         <img
                             src="{IMG_URL}{provider.logo_path}"
                             alt={provider.provider_name}
                         />
                     </div>
-                </div>
-            {/each}
-        </div>
+                {/each}
+            </div>
+        {:else if !providers.length}
+            <div class="pt-5">
+                <div class="badge badge-lg">No providers</div>
+            </div>
+        {:else}
```

Comment:
The if/elif/else order hurts my brain, mby 1. not, 2. more, 3. less? or reverse

Reply:
Done.